index.html

```diff
@@ -74,6 +98,12 @@ <h1 id="header">Basics: Beat That! 🚀</h1>
 
         // Reset input value
         input.value = "";
+
+        // Hide game instructions text once the game starts
+        var instructions = document.querySelectorAll(".instruction-text");
+        instructions.forEach(function (instruction) {
+          instruction.classList.add("hidden-text");
+        });
```

Comment:
Awesome job on doing this! You do know how to take in a tag with the `querySelector` and hide it! I don't particularly like the `array.forEach()` function because it's a bit finicky, but this works!

p.s. I'd like to use a for loop to iterate through this
```
for(var i = 0; i<instructions.length; i+=1){
  instructions[i].classList.add("hidden-text")
}
```